src/main/java/sx/blah/discord/handle/impl/obj/Guild.java

```diff
@@ -847,6 +845,23 @@ public IEmoji getEmojiByName(String name) {
 	}
 
 	@Override
+	public IEmoji createEmoji(String name, Image image, IRole... roles) {
+		if (!DiscordUtils.EMOJI_NAME_PATTERN.matcher(name).find())
```

Comment:
Why is this checked differently from other name checks?

Reply:
What exactly is different about it and what is it different from?